packages/react-components/src/components/assets/AttachmentCardIcon.tsx

```diff
@@ -0,0 +1,254 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+
+/**
+ * docx icon for automation tests
```

Comment:
If these are for tests, should we have the files as part of the test code, rather than part of the components?

Reply:
Agreed. I think I could also be in the above `index.tsx` since its dedicated to playwrite is my suggestion

Reply:
moved it under playright folder